src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
move to * selector

Reply:
not fixed 
move box-sizing to * selector and also font-family to html selector